Doc/reference/ldap.rst

```diff
@@ -1068,7 +1068,10 @@ and wait for and return with the server's result, or with
    or :py:meth:`search_ext_s()` (client-side search limit). If non-zero
    not more than *sizelimit* results are returned by the server.
 
+   .. versionchanged:: 3.0
 
+      ``filterstr=None`` is equivalent to ``filterstr='(objectClass=*)'``.
+   
```

Comment:
Looks good; the only nitpick I found is to say `equivalent` instead of `equal` here.